src/assets/styles/styles/custom/_button.scss

```diff
@@ -52,3 +52,29 @@
   display: flex;
   justify-content: center;
 }
+
+.btn-dropdown {
+  border-radius: 0px;
+  &:hover {
+    cursor: pointer;
+    font-weight: 600;
+    box-shadow: 0px 0px;
+  }
+  float: left;
+  clear: left;
+  padding: 5px 7px 5px;
+  line-height: 20px;
+}
+
+.dropdown-buttons-container {
```

Comment:
Ja bym wyniósł to do osobnego pliku, ponieważ ten dropdown raczej nie jest powiązany z przyciskami. 
Wtedy można by było go nazwać jakoś w stylu `_dropdown.scss`.
I potrzeba będzie też zaimportować ten plik w `style-all.scss`, podobnie jak to jest robione tutaj
https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/blob/445796a22ff0f270917e8fe0491e48e185fc5874/src/assets/styles/styles-all.scss#L23-L34